src/server/tests.rs

```diff
@@ -1437,6 +1507,84 @@ x + 1
     );
 }
 
+#[test]
+async fn test_hover_with_markdown() {
+    let fluxscript = r#"x = 1
+x + 1
+"#;
+    let server = create_server();
+    let params = lsp::InitializeParams {
+        capabilities: lsp::ClientCapabilities {
+            workspace: None,
+            text_document: Some(
+                lsp::TextDocumentClientCapabilities {
+                    synchronization: None,
+                    completion: None,
+                    hover: Some(lsp::HoverClientCapabilities {
+                        dynamic_registration: Some(true),
+                        content_format: Some(vec![
+                            lsp::MarkupKind::Markdown,
+                        ]),
```

Comment:
Could we add a test for when `content_format` is a `Vec` that has both `MarkupKind` variants in it? That _looks_ like a valid case that could arise. If `Markdown` is supported, we should always prefer it.